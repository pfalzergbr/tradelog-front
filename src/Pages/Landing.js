import React from 'react';

const Landing = (props) => {
    return <div>Landing Page</div>;
};

export default Landing;
