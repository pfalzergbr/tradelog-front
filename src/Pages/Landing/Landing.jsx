import React from 'react';

import Footer from '../Shared/Footer';

const Landing = () => {
  return (
    <div className='landing-page'>
      <div className='landing-page__container'>
        <main className='landing-page__main'>
          <h1 className='landing-page__title'>Keep track of your trades. Find your edge. Grow your profits</h1>
          <h3 className='landing-page__subtitle'>
            Analyse your accounts. Find your most profitable trading strategies.
            Repeat.
          </h3>
          <button className='btn btn--primary'>Sign up for free</button>
        </main>
        <div className='landing-page__graphic-column'>
          <div className='graphic'>
            <object
              className='graphic--background'
              type='image/svg+xml'
              data='./svg/icon-group.svg'></object>
            <object
              className='graphic--front'
              type='image/svg+xml'
              data='./svg/icon-group2.svg'></object>
          </div>
        </div>
      </div>
      <Footer />
    </div>
  );
};

export default Landing;
