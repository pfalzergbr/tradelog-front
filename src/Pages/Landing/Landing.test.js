import { shallow } from 'enzyme';
import Landing from './Landing';
import React from 'react';

it('should render Landing Component', () => {
    expect(shallow(<Landing />)).toMatchSnapshot();
})