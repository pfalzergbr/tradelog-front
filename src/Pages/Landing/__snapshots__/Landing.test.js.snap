// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Landing Component 1`] = `
<div
  className="landing-page"
>
  <main
    className="landing-page__main"
  >
    Track your trades. Find your edge. Grow your profits.
  </main>
  <Footer />
  ;
</div>
`;
